lib/models.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
```

Comment:
I think it's better to split into multiple file, could you research if there is a better way than assign every single property manually?